CHANGELOG.md

```diff
@@ -6,8 +6,31 @@ Deprecated:
 - Authorization using Role-Based Access Control
 - LDAP & OIDC Embedded Support
```

Comment:
isn't this covered by the changelog below?

Reply:
OIDC is, LDAP not yet.  Removing "OIDC".